WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/Coupon Text Validators/CouponCodeInputFormatter.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+/// `UnitInputFormatter` implementation for Coupon Code input.
+///
+struct CouponCodeInputFormatter: UnitInputFormatter {
+    func isValid(input: String) -> Bool {
+        // Every string is a valid coupon code.
+        return true
+    }
+
+    func format(input text: String?) -> String {
+        return text?.uppercased() ?? ""
```

Comment:
What do you think about avoiding return statements, when we have a single statement in a function?